azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -515,8 +515,17 @@ private void startWebMetrics() throws Exception {
     // (N is configurable by MIN_AGE_FOR_CLASSIFYING_A_FLOW_AGED_MINUTES).
     // ToDo(anish-mal) Enable this for push based dispatch logic.
     if (this.props.getBoolean(ConfigurationKeys.AZKABAN_POLL_MODEL, false)) {
-      this.metricsManager
-          .addGauge("WEB-NumAgedQueuedFlows", this.executorManagerAdapter::getAgedQueuedFlowSize);
+      int minAge =
+          this.props.getInt(ConfigurationKeys.MIN_AGE_FOR_CLASSIFYING_A_FLOW_AGED_MINUTES,
+              Constants.DEFAULT_MIN_AGE_FOR_CLASSIFYING_A_FLOW_AGED_MINUTES);
+      if (minAge >= 0) {
```

Comment:
Would have preferred this check in the Dao so that any other future users of the method are subject to the same check. 
However likelihood of running into this is low and given that you are planning subsequent changes to this, I am fine with retaining it here.